service/host.go

```diff
@@ -164,6 +164,7 @@ func (s *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	// Flush tracing buffers
+	// GFM-378 uhttp.Client is not safe to send requests anymore
```

Comment:
yes but they are not usually aware of this at all. They don't *know* that this is how it works and they don't have. UberFx's thing is all this functionality works only if you initialize and start the service.